app/javascript/components/admin/AuditModal.js

```diff
@@ -0,0 +1,338 @@
+          );
+        } else {
+          return (
+            <span>
+              <b>Account Status</b>: Locked
+            </span>
+          );
+        }
+      case 'jurisdiction_id':
+        if (!change.details || !Array.isArray(change.details) || change.details.length < 2) {
+          return (
+            <span>
+              <b>Jurisdiction</b>: Updated
+            </span>
+          );
+        } else {
+          return (
+            <span>
+              <b>Jurisdiction</b>: Changed from &quot;{_.invert(this.state.all_jurisdiction_paths)[change.details[0]]}&quot; to &quot;
+              {_.invert(this.state.all_jurisdiction_paths)[change.details[1]]}&quot;
+            </span>
+          );
+        }
+      case 'created_at':
+        return (
+          <span>
+            <b>Account Created</b>
+          </span>
+        );
+      case 'api_enabled':
+        if (!change.details || !Array.isArray(change.details) || !change.details.length) {
+          return (
+            <span>
+              <b>API Access</b>: Updated
+            </span>
+          );
+        } else if (change.details[0]) {
+          return (
+            <span>
+              <b>API Access</b>: Disabled
+            </span>
+          );
+        } else {
+          return (
+            <span>
+              <b>API Access</b>: Enabled
+            </span>
+          );
+        }
+      case 'role':
+        if (!change.details || !Array.isArray(change.details) || change.details.length < 2) {
+          return (
+            <span>
+              <b>Role</b>: Updated
+            </span>
+          );
+        } else {
+          return (
+            <span>
+              <b>Role</b>: Changed from &quot;{change.details[0]}&quot; to &quot;{change.details[1]}&quot;
+            </span>
+          );
+        }
+      case 'email':
+        if (!change.details || !Array.isArray(change.details) || change.details.length < 2) {
+          return (
+            <span>
+              <b>Email</b>: Updated
+            </span>
+          );
+        } else {
+          return (
+            <span>
+              <b>Email</b>: Changed from &quot;{change.details[0]}&quot; to &quot;{change.details[1]}&quot;
+            </span>
+          );
+        }
+      case 'authy_enabled':
+        if (!change.details || !Array.isArray(change.details) || !change.details.length) {
+          return (
+            <span>
+              <b>2FA</b>: Updated
+            </span>
+          );
+        } else if (change.details[0]) {
+          return (
+            <span>
+              <b>2FA</b>: Enabled
+            </span>
+          );
+        } else {
+          return (
+            <span>
+              <b>2FA</b>: Disabled
+            </span>
+          );
+        }
+      case 'force_password_change':
+        return (
+          <span>
+            <b>Password Changed/Reset</b>
+          </span>
+        );
+      case 'last_sign_in_with_authy':
```

Comment:
This is a change that is triggered by the user themselves and not any admins. Any reason in particular we decided to track logins in the audit history? @holmesie?

Reply:
I know there was at least one user who expressed strong interest in having this particular information available in the audit log.

Reply:
That's enough for me!